rskj-core/src/test/java/co/rsk/peg/whitelist/WhitelistSupportImplTest.java

```diff
@@ -104,6 +109,19 @@ void getLockWhitelistEntryByAddress_whenLockWhitelistHasEntries_shouldReturnOneO
         assertEquals(btcAddress, actualLockWhitelistEntry.address());
     }
 
+    private void saveInMemoryStorageOneOffWhiteListEntry() {
+        Coin maxTransferValue = Coin.COIN;
+        OneOffWhiteListEntry oneOffWhiteListEntry = new OneOffWhiteListEntry(btcAddress, maxTransferValue);
+        List<OneOffWhiteListEntry> oneOffWhiteListEntries = Collections.singletonList(oneOffWhiteListEntry);
+        Pair<List<OneOffWhiteListEntry>, Integer> pairValue = Pair.of(oneOffWhiteListEntries, 100);
```

Comment:
What is `100`? Maybe put in a variable so it's easy to understand what it represents

Reply:
Done! The Magic number was replaced with a meaningful constant.